### PR TITLE
docs(cp): Add new admin Task API docs

### PR DESCRIPTION
**Pull request by bucanero**
Fix DOC-318

### CODE REVIEW

File: content/deploy/dgraph-administration.md [hunk 1]
```diff
@@ -152,6 +150,13 @@ mutation {
   }
 }
 ```
+
+{{% notice "note" %}}
+Since v21.03, the `export` and `backup` APIs are asynchronous: instead of returning the requested data,
```
Comment (ajeetdsouza) [inline content/deploy/dgraph-administration.md:155]: Only backups are async in 21.03. Exports will be made async in the next release.

File: content/deploy/dgraph-administration.md [hunk 2]
```diff
@@ -166,6 +171,23 @@ cluster. Dgraph does not copy all files to the Alpha that initiated the export.
 The user must also ensure that there is sufficient space on disk to store the
 export.
 
+### Check queued tasks
+
+A new Task API has been added to the `/admin` endpoint. This allows you to check the status of a queued task (either `backup` or `export`).
```
Comment (ajeetdsouza) [inline content/deploy/dgraph-administration.md:176]: No export.

File: content/graphql/admin/index.md [hunk 3]
```diff
@@ -235,14 +257,17 @@ Here are the important types, queries, and mutations from the `admin` schema.
...
+	}
 	type Response {
 		code: String
 		message: String
 	}
 
 	type ExportPayload {
 		response: Response
-		exportedFiles: [String]
+		taskId: String
```
Comment (ajeetdsouza) [inline content/graphql/admin/index.md:270]: Remove taskId, add exportedFiles back.

Comment (bucanero) [reply content/graphql/admin/index.md]: Fixed

File: content/deploy/dgraph-administration.md (hunk 2 above)
Comment (bucanero) [reply content/deploy/dgraph-administration.md]: Fixed

File: content/deploy/dgraph-administration.md (hunk 1 above)
Comment (bucanero) [reply content/deploy/dgraph-administration.md]: Fixed 👍

**Pull request merged**